snowplow/iglu-client-embedded/schemas/com.metabase/setup/jsonschema/1-0-3

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Setup flow",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "setup",
+    "format": "jsonschema",
+    "version": "1-0-2"
+  },
+  "type": "object",
+  "properties": {
+    "event": {
+      "description": "Event name",
+      "type": "string",
+      "enum": [
+        "step_seen",
+        "usage_reason_selected",
+        "database_selected",
+        "add_data_later_clicked"
+      ],
+      "maxLength": 1024
+    },
+    "version": {
+      "description": "String describing the version of onboarding we're on",
+      "type": "string",
+      "maxLength": 1024
+    },
+    "step": {
+      "description": "String describing the step that the set up step",
+      "type": [
+        "string",
+        "null"
+      ],
+      "enum": [
```

Comment:
https://github.com/metabase/metabase/blob/312869f0d9eeb17b61789d31346a18fd9f970b91/frontend/src/metabase/setup/types.ts#L1-L10

From the type, is doesn't have step name which is `null`, so I don't think we need to add `null` here.

Reply:
Actually reading [the docs here](https://json-schema.org/understanding-json-schema/reference/null) it may be that we don't even need the first null (in `type`) as we never pass it explicitly as null and it's not marked as required.
To keep the schema backward compatible I wouldn't remove it though